### PR TITLE
Release highlights and change log review for 2022.1

### PR DESCRIPTION
**Pull request by seanbudd**
Add release highlights for the 2022.1 release.

**(squash merge)**

### CODE REVIEW

**Review by Qchristensen — DISMISSED** [review]
All looks good, great work!

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -4,11 +4,24 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2022.1 =
+This release includes major improvements to UIA support with MS Office.
+For Microsoft Office 16.0.15000 and higher on Windows 11, NVDA will use UI Automation to access Microsoft Word documents by default.
+This provides a significant performance improvement over the old Object model access.
+
+There are improvements to braille display drivers including Seika Notetaker, Papenmeier and HID Braille. 
+There are also various Windows 11 bug fixes, such as Calculator, Console, Terminal, Mail and Emoji Panel.
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:12]:
```suggestion
There are also various Windows 11 bug fixes, for apps such as Calculator, Console, Terminal, Mail and Emoji Panel.
```

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -62,10 +75,10 @@ What's New in NVDA
 - In Windows 10 Calculator, NVDA will announce labels for history and memory list items. (#11858)
 - Gestures such as scrolling and routing again work with HID Braille devices. (#13228)
 - Windows 11 Mail: After switching focus between apps, while reading a long email, NVDA would get stuck on a line of the email. (#13050)
-- HID braille: chorded gestures (E.g. space+dot4) can be successfully performed from the Braille display. (#13326)
+- HID braille: chorded gestures (e.g. space+dot4) can be successfully performed from the Braille display. (#13326)
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:78]:
```suggestion
- HID braille: chorded gestures (e.g. ``space+dot4``) can be successfully performed from the Braille display. (#13326)
```

**Review by feerrenrut — DISMISSED** [review]

File: user_docs/en/changes.t2t [hunk 3]
```diff
@@ -42,9 +55,9 @@ What's New in NVDA
 - ``NVDA+Numpad Delete`` reports the location of the caret or focused object by default. (#13060)
 - ``NVDA+Shift+Numpad Delete`` reports the location of the review cursor. (#13060)
 - Added default bindings for toggling modifier keys to Freedom Scientific displays (#13152)
-- 'Baseline' is no longer reported via the report text formatting command (NVDA+F). (#11815)
+- "Baseline" is no longer reported via the report text formatting command (``NVDA+f``). (#11815)
```
Comment (CyrilleB79) [inline user_docs/en/changes.t2t:58]: NVDA+F shortcut styling has been modified here:
- Code styling (fixed width) is used (adding double backquotes around the expression)
- The F has been switched from uppercase to lowercase.
Is it general styling recommendations to write documentation? Or is it just a matter of personal preference?

I have no strong reason to advocate for one styling or the other, but it's just to know if I need to change something in the future when writing documentation.

Thanks.

**Comment by CyrilleB79** [comment]
There is an error in the initial description of this PR:
2021.3 -> 2022.1
Even if the title of the PR is OK, please fix it for clarity.

File: user_docs/en/changes.t2t [hunk 4]
```diff
@@ -62,10 +75,10 @@ What's New in NVDA
 - In Windows 10 Calculator, NVDA will announce labels for history and memory list items. (#11858)
 - Gestures such as scrolling and routing again work with HID Braille devices. (#13228)
 - Windows 11 Mail: After switching focus between apps, while reading a long email, NVDA would get stuck on a line of the email. (#13050)
```
Comment (lukaszgo1) [inline user_docs/en/changes.t2t:77]: This needs to be reworded. What about:
```suggestion
- Windows 11 Mail: After switching focus between apps, while reading a long email, NVDA would no longer get stuck on a line of the email. (#13050)
```

Comment (seanbudd) [reply user_docs/en/changes.t2t:77]: I think this reads better:

```suggestion
- Windows 11 Mail: After switching focus between apps, while reading a long email, NVDA no longer gets stuck on a line of the email. (#13050)
```

File: user_docs/en/changes.t2t (hunk 3 above)
Comment (seanbudd) [reply user_docs/en/changes.t2t:58]: >Code styling (fixed width) is used (adding double backquotes around the expression)
I think we are trying to do this consistently now.
If this formatting is missing from other entries for this release and future releases, we should try to update them.
I don't know if this convention has been documented anywhere - they are a bit of a developing standard.

>The F has been switched from uppercase to lowercase.

When documenting keyboard commands, "f" is the key name by convention, "F" tends to imply "shift+f".

**Comment by seanbudd** [comment]
I've reviewed the changes.html file generated, and everything looks good. I have one remaining issue with the changelog:

Should the "arrow" -> be replaced with "then, " in the following entry:

```t2t ​  - MathType equations must also be manually converted to Office Math by selecting each and choosing Equation options -> Convert to Office Math in the context menu. 
```

If this isn't an issue, can this be merged in to beta?

**Comment by feerrenrut** [comment]
> Should the "arrow" -> be replaced with "then, " in the following entry:

Yes, some other ways to address this would be:
- put the steps on separate lines as sub-points/bullets
- use just a comma: `manually converted to Office Math by selecting each, opening the context menu, choosing Equation options, Convert to Office Math. `

**Comment by seanbudd** [comment]
> - use just a comma: `manually converted to Office Math by selecting each, opening the context menu, choosing Equation options, Convert to Office Math. `


I'm going to use this option

**Review by feerrenrut — APPROVED** [review]

**Comment by josephsl** [comment]
Hi, a suggestion: in the beginning of the changes for devs section in 2022.1, there appears to be a grammatical error – “a add-on” -> “ an add-on”. Thanks.

**Comment by seanbudd** [comment]
@josephsl 2022.1 was released a long time ago. We do not update the change log section of previous releases.

**Pull request merged**